contact.html

```diff
@@ -0,0 +1,129 @@
+  <meta name="msapplication-config" content="/resources/favicons/browserconfig.xml">
+  <meta name="theme-color" content="#ffffff">
+</head>
+<body>
+  
+  <header>
+    <div class="container flex-box head-box">
+      <h1><a href="#"><img src="resources/images/logo.jpg" alt="Rmedia"></a></h1>
+      <input type="search" name="search" id="search" placeholder="search...">
+    </div>
+  </header>
+
+  <section class="content">
+
+    <div class="container">
+
+      <div class="content-section flex-box">
+        <section class="left-section">
+          <div class="drop-down">
+            <button class="drop-down-button">Menu</button>
+          </div>
+          <nav class="navigation">
+            <ul id="nav-menues">
+              
+            </ul>
+          </nav>
+        </section>
+        <section class="middle-section">
+          <div class="top-section">
+            <div class="content-area">
+              <h2>Contact</h2>
+            </div>
+          </div>
+          <div class="bottom-section">
+            <form class="form">  
+
+              <!-- Name Field -->
+              <div class="label-box flex-box">
+                <label for="name">Name</label>
+                <p id="name-counter">25</p>
+              </div>
+              <input type="text" name="name" class="input" id="name" maxlength="25" required>
+              <p class="label" id="name-error"></p>
+              
+              <!-- Subject Field -->
+              <div class="label-box flex-box">
+                <label for="Subject">Subject (Not Required)</label>
+                <p id="subject-counter">25</p>
+              </div>         
+              <input type="text" class="input" name="Subject" id="subject" maxlength="25">
+              <p class="label" id="subject-error"></p>
+              
+              <!-- Phone Field -->
+              <div class="label-box flex-box">
+                <label for="Phone">Phone</label>
+              </div>
+              <input type="text" name="Phone" class="input" id="phone" maxlength="10" required>
+              <p class="label" id="phone-error"></p>
+              
+              <!-- Email Field -->
+              <div class="label-box flex-box">
+                <label for="Email">Email</label>
+              </div>
+              <input type="email" name="Email" class="input" id="email" required>
+              <p class="label" id="email-error"></p>
+
+              <!-- Description Field -->
+              <div class="label-box flex-box">
+                <Label>Description</Label>
+                <p id="message-counter">255</p>
+              </div>
+              <textarea name="Message" class="input" id="message" maxlength="255" required></textarea>
+              <p id="message-error" class="label"></p>
+              
+              <!-- Button -->
+              <button type="button" class="formBtn">SUBMIT</button>
+            </form>
```

Comment:
@Thejus5 A field in the form is a group of below items:
> -label-
> -input field-
> -error-
Hence it is ideal to group them under a div(if u r looking for block element) or sections.